src/index.ts

```diff
@@ -25,11 +25,7 @@ const rules = readdirSync(rulesDir)
   );
 
 const allRules = Object.keys(rules).reduce<Record<string, string>>(
-  (rules, key) => {
-    rules[`jest/${key}`] = 'error';
-
-    return rules;
-  },
+  (rules, key) => ({ ...rules, [`jest/${key}`]: 'error' }),
```

Comment:
This is actually surprisingly "slow", b/c everytime you're creating a new object & copying all the keys.

It's by far not a big deal, and even less so since we're doing it the once (I hope? I assume the way eslint runs it typically reuses the same node process), but it's an interesting little fact.

(That might have changed in later versions of node - iirc I last tested this on node8)

Reply:
doing it once during `require`, yeah - probably not even measurable except in micro benchmarks. _Nothing_ compared to the FS operations in the same file 🙂

Reply:
and yeah, object spread has gotten way quicker (although it's currently transpiled down by babel until we drop node 6)

https://twitter.com/bmeurer/status/1015105293301280768?lang=en